packages/core/src/components/text-field/text-field.tsx

```diff
@@ -130,6 +130,15 @@ export class TdsTextField {
     this.tdsBlur.emit(event);
   }
 
+  /** Method to handle focus */
+  @Method()
+  async focusTextField() {
```

Comment:
Maybe we should try and replicate the focus method of an `<input/>` ?

```suggestion
  async focus() {
```

Reply:
I’ve already tried that, but I’m getting this warning. What are your thoughts? @mJarsater 

> The @Method() name "focus" is a reserved public name. Please rename the "focus" method so it does not conflict with an existing standardized prototype member. Reusing method names that are already defined on the element's prototype may cause unexpected runtime errors or user-interface issues on various browsers, so it's best to avoid them entirely.

Reply:
Ah! Yeah I see, let's leave it at `focusTextField ` then for now